lib/persistence/object.rb

```diff
@@ -312,12 +312,10 @@ def get_notifications
 
       notification_files.inject('processed' => [], 'failed' => []) do |notifications, s3_object|
         _, _, filename  = s3_object.key.split('/')
-        _, status, object_type, object_ref, _ = filename.split('_')
+        _, status, object_type, _object_ref, _ = filename.split('_')
```

Comment:
Is `_object_ref` intentional? You call for `object_ref` (no underscore at the beginning) a few lines down.

Reply:
Yeah - I thought setting object_ref here wasn't doing anything, so I added an underscore (not sure if that's the way to say "I'm not using this") Really, we set the object_ref below.

The two assignments aren't related and the second one is the right one.

Reply:
The underscore is the way to say "throw this value out".